### ISSUE TITLE
[BUG] NEO The World Ends With You touch pad issue

### ISSUE THREAD

**Issue by BortzANATOR** [streetpea/chiaki-ng#454]
**Describe the bug**
I have found an issue while playing NEO: The World Ends With You PS4 through Chiaki-ng. Using "T" to activate a touch pad press does not work in this specific game. "T" to act as a touch pad button press to bring up a menu works in other games like Bugsnaks. But curiously the same command does not work specifically in NEO: TWEWY. I have tried many many different combinations of settings through the Steam Input settings. Mouse click, button press, trigger... They all work in other games (that I have played) but this is the first time I can't activate the menu through a "T" button press. 

**To Reproduce**
Make sure you have a button mapped to act as a button press of the PS4 touch pad. 
Remote connect to the PS4 
Boot NEO: TWEWY 
Use corresponding button to try and bring up the difficulty menu in game. 

**Expected behavior**
"T" (or whatever you set touchpad to in the settings) should act as a touch pad button press.

**Comment by streetpea** [comment]
@BortzANATOR this sounds like a remote play bug with TWEWY not a chiaki-ng bug… can you try via the official remote play app and see if it fails there?

**Comment by BortzANATOR** [comment]
Ah ok, maybe it is! I don't have any other devices set up at the moment to take advantage of the official remote play app. I might need a few days before I can get around to a working setup. 

That would explain the perplexing nature of the problem. The only other thing I could think of is that the game is looking for a different kind of input for the menu? Left click or both or looking for movement too? I have no idea.

**Comment by streetpea** [comment]
@BortzANATOR unless it’s specifically a left touchpad click or something (which you can use dpad touch emulation for as of 1.9.1) it’s likely just a bug when using remote play. I believe re4 remake had something similar and it was fixed with a patch.

**Comment by streetpea** [comment]
@BortzANATOR if you think it’s something weird going on with using a touchpad click vs a touchpad you could always plug in a dualsense or DualShock 4 (disable Steam input for the controller if playing via Steam), and test directly with the touchpad using chiaki-ng as well.

**Comment by streetpea** [comment]
I don’t think this is something chiaki-ng can fix since it seems like a game specific remote play bug so I’m closing this.
